spec/models/carriers/usps_calculator_spec.rb

```diff
@@ -0,0 +1,95 @@
+require 'spec_helper'
+
+module ActiveShipping
+  describe Spree::Calculator::Shipping::Usps do
+    WebMock.disable_net_connect!
```

Comment:
Is this usually enabled? If so do we need to turn it back on after tests?

Either way we should probably do this in a before